coremltools/converters/keras/_layers2.py

```diff
@@ -199,6 +201,23 @@ def convert_activation(builder, layer, input_names, output_names, keras_layer):
         builder.add_softmax(name = layer, input_name = input_name,
                 output_name = output_name)
         return
+    if non_linearity == 'RELU6':
+        # No direct support of RELU with max-activation value - use negate and 
```

Comment:
Do you mean "RELU6" rather than just "RELU"?

Reply:
I have the random doubt that this particular layer name is in lowercase in keras